exodus/reports/tests.py

```diff
@@ -1,6 +1,57 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
-# from django.test import TestCase
+from django.test import TestCase, Client
 
-# Create your tests here.
+from reports.models import Report, Application
+
+
+class ReportsViewTests(TestCase):
+    REPORTS_PATH = '/en/reports/'
+
+    def test_should_return_reports_count_with_2_reports(self):
+        report1 = Report(id=1)
+        report2 = Report(id=2)
+
+        report1.save()
+        report2.save()
+
+        c = Client()
+        response = c.get(self.REPORTS_PATH)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.context['reports_count'], 2)
+
+    def test_should_return_apps_count_with_2_applications(self):
+        report1 = Report(id=1)
+        report2 = Report(id=2)
+        report1.save()
+        report2.save()
+
+        app1 = Application(id=1, name="App1", report=report1, handle="com.test.track1")
+        app2 = Application(id=2, name="App2", report=report2, handle="com.test.track2")
+        app1.save()
+        app2.save()
```

Comment:
Similarly, you can write:
```
report1 = Report.objects.create()
report2 = Report.objects.create()
Application.objects.create(name='App1', report=report1, handle='com.test.track1')
Application.objects.create(name='App2', report=report2, handle='com.test.track2')
```

Not sure if you need to set the ids.

Reply:
Correct thx